chia/cmds/init_funcs.py

```diff
@@ -381,6 +400,18 @@ def chia_init(
         fix_ssl(root_path)
     if should_check_keys:
         check_keys(root_path)
+    if experimental_v2_db:
+        config: Dict = load_config(root_path, "config.yaml")["full_node"]
+        db_path_replaced: str = config["database_path"].replace("CHALLENGE", config["selected_network"])
```

Comment:
I noticed a similar pattern is used in `full_node.py` and `wallet_node.py`. Just something to consider, but if there's enough code to warrant it, it might make sense to create a new file to contain "db init" functionality.